android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,7 +117,38 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle = "";
+                    String track = "";
+                    String color = null;
```

Comment:
@iamareebjamal They need to be declared before because they are being used inside the loop. As I said earlier, it gives an error if I don't initialise them.